packages/service-library/tests/data/oas3/enveloped_responses.yaml

```diff
@@ -102,6 +102,7 @@ components:
       properties:
         data:
           type: string
+          format: password
```

Comment:
we also had some parts with format: uuid that was not working I think. maybe now it does...